rules/test-title.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var util = require('../util');
+var literalTypes = ['Literal', 'TemplateLiteral'];
+
+module.exports = function (context) {
+	var ifMultiple = context.options[0] === 'if-multiple';
+	var isTestFile = false;
+	var testCount = 0;
+
+	return {
+		CallExpression: function (node) {
+			if (util.isTestFile(node)) {
+				isTestFile = true;
+				return;
+			}
+
+			if (isTestFile) {
+				var args = node.arguments;
+				var callee = node.callee;
+				var name = callee.type === 'MemberExpression' ? callee.object.name : callee.name;
+				if (name === 'test') {
+					testCount++;
+					var hasNoTitle = args.length !== 2 || literalTypes.indexOf(args[0].type) === -1;
```

Comment:
Yeah, it seemed nice when I added it, but it's not very useful and creates inconsistency. Maybe we should just deprecate it in AVA (?). Hmm. Maybe hold off on supporting this pattern for now, until we decide on what to do about it. I don't want it holding this up.